docs/guides/setup-channel.mdx

```diff
@@ -49,55 +48,47 @@ Available plans:
 
     ```python
     from qiskit_ibm_runtime import QiskitRuntimeService
 
-    # Save an IBM Quantum account and set it as your default account.
     QiskitRuntimeService.save_account(
-        channel="ibm_quantum",
-        token="<MY_IBM_QUANTUM_TOKEN>",
-        set_as_default=True,
-        # Use `overwrite=True` if you're updating your token.
-        overwrite=True,
+      token="<your-token">,
+      channel="ibm_quantum", # `channel` distinguishes between different account types
+      overwrite=True # Use `overwrite=True` if you're updating your token
     )
+    ```
+
+    Close out of the REPL with `exit()`. From now on, whenever you need to authenticate to the service, you can load your credentials with `QiskitRuntimeService()`.
 
+    ```python
     # Load saved credentials
     service = QiskitRuntimeService()
     ```
 
-    * If you save your credentials to disk, you can use `QiskitRuntimeService()` in the future to initialize your account. The `channel` parameter distinguishes between different account types.
-    * If you are saving multiple accounts per channel, consider using the `name` parameter to differentiate them.
-    * Credentials are saved to `$HOME/.qiskit/qiskit-ibm.json`.  Do not manually edit this file.
-    * If you don't save your credentials, you must specify them every time you start a new session.
-    * The `channel` parameter allows you to distinguish between different account types. When initializing the account, IBM Cloud is the default account used if have saved credentials for an IBM Quantum Platform and an IBM Cloud account.
+      * If you have saved credentials for both an IBM Quantum Platform account and an IBM Cloud account, IBM Cloud is the default account used when you initialize your account, unless you specify `set_as_default=True` in your IBM Quantum Platform account when you use the `save_account()` method.
+      * If you are saving multiple accounts per channel, consider using the `name` parameter to differentiate them.
+      * Credentials are saved to `$HOME/.qiskit/qiskit-ibm.json`.  Do not manually edit this file.
 
-    <Admonition type="caution">
-      Account credentials are saved in plain text, so only do so if you are using a trusted device.
+    <Admonition type="warning">
+      **Protect your API token!** Never include your token in plain text in a notebook file. If you are working in a trusted Python environment (such as on a personal laptop or workstation), use the `save_account()` method described in the previous step to store your credentials, keeping in mind that your token will still be stored as plain text on your local drive. If you are executing code in a Python environment that is not trusted, see the [instructions in the next step](#untrusted-machine).
```

Comment:
This warning should be higher up - e.g. before any code snippets. Otherwise people would likely copy&paste the code into their notebook before getting to this warning.

Reply:
done in 0e18cd37b89489d3fcfb0d38a41d5163e9134178